landoui/app.py

```diff
@@ -9,6 +9,10 @@
 from flask_assets import Environment
 from webassets.loaders import YAMLLoader
 
+from landoui import auth
+
+oidc = None
```

Comment:
It's global because it is a variable that is initialized at run-time for the Flask app, but, also needed at import time for the different Blueprints so that the decorators in those files can work properly.